### PR TITLE
feat: Add Cosmas package

### PR DESCRIPTION
**Pull request by RaniaZeid**

### CODE REVIEW

**Comment by RaniaZeid** [comment]
@ahmdyasser  @ahmednaagy  @oradwanomar  @claramounir  @HebaMad  review please

**Review by oradwanomar — APPROVED** [review]

**Review by ahmdmhasn — CHANGES_REQUESTED** [review]
It should be added to UIDokan.. and we need to add a wrapper to it.

File: Cosmas/Package.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Cosmas",
+            targets: ["Cosmas"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/evgenyneu/Cosmos.git",.upToNextMajor(from: "23.0")
```
Comment (ahmdmhasn) [inline Cosmas/Package.swift]: ya Rania it should be added to UIDokan not new one

**Review by ahmdmhasn — CHANGES_REQUESTED** [review]

**Comment by RaniaZeid** [comment]
another one open #105

**Pull request closed without merging**